specs/minimal_init.vim

```diff
@@ -0,0 +1,25 @@
+set rtp+=.
+set rtp+=./vendor
+set noswapfile
+
+let $lsp_format = getcwd()
+let $specs = getcwd() .. "/specs"
+let $vendor = getcwd() .. "/vendor"
+
+set rtp+=$lsp_format,$specs
+set packpath=$vendor
+
+packloadall
+
+lua <<EOF
+	require 'lsp-format'.setup {}
+	require 'lspconfig'.denols.setup { on_attach = require 'lsp-format'.on_attach }
+EOF
+
+function! RunSpecs() abort
+	lua <<EOF
+    require("plenary.test_harness").test_directory(os.getenv("FILE") or "./specs", {
```

Comment:
@lukas-reineke Is this not the plenary test harness already?

Reply:
yes, but you make it way more complicated than it needs to be.
you only need `nvim --headless -c "PlenaryBustedDirectory tests/plenary/ {minimal_init = 'tests/minimal_init.vim'}"` in the make file and the test file.